css/style.css

```diff
@@ -0,0 +1,150 @@
+#top {
+  background-color: var(--color-red);
+}
+
+img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.footer-social p {
+  font-size: 12px;
+}
+
+.footer-ul {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.footer-logo {
+  object-fit: unset;
+}
+
+#top img {
+  object-fit: unset;
+}
```

Comment:
Group selectors with the same style definition

Reply:
ok thanks, it's done